packages/frontend/src/components/RichTextEditor/index.tsx

```diff
@@ -84,7 +84,7 @@ const Editor = ({
       }),
       StepVariable,
     ],
-    content: initialValue,
+    content: substituteOldTemplates(initialValue, varInfo), // back-ward compatibility with old values from PowerInput
```

Comment:
We also need to convert newlines into `<br />`

Reply:
Only needs this if we're reapplying to the non-richtext inputs, implemented in #278